test/e2e/test-apps/javascript/main-error/event.json

```diff
@@ -69,6 +69,7 @@
                 "colno": 0,
                 "filename": "app:///src/main.js",
                 "function": "{{function}}",
+                "module": "main",
```

Comment:
Added this check in here. If it passes on all platforms we're all good!

Reply:
![image](https://user-images.githubusercontent.com/18689448/182387174-4b255193-2a5d-4ec7-ae4d-34535035d29b.png)

(bringing the :salute_canada: emoji react to GH to share with the world :p)